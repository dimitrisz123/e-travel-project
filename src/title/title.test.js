import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import Title from "./title";

it("renders Title correctly", () => {
	const wrapper = shallow(<Title />);
	expect(toJson(wrapper)).toMatchSnapshot();
});
