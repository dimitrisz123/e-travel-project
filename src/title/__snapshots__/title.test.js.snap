// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Title correctly 1`] = `
<button
  className="title"
  onClick={[Function]}
>
  <div
    className="title-inner"
  >
    <span>
      EPISODE: 
    </span>
    <span
      className="movie-name"
    />
    <span />
  </div>
</button>
`;
